examples_apps/run_tutorial_similarity.py

```diff
@@ -1,314 +0,0 @@
-# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports,invalid-name

```

Comment:
One thing to bear in mind is that https://github.com/XanaduAI/strawberryfields/pull/390 wants to change this tutorial :thinking: I guess we can just do that in a separate PR to photonic-applications?

Reply:
Yep!